packages/shared-influx-pieces/src/register.ts

```diff
@@ -0,0 +1,6 @@
+import { container } from '@skyra/http-framework';
+import { registerSchedule } from 'lib/utils/schedule.js';
+
+container.stores.registerPath(new URL('.', import.meta.url));
+
+registerSchedule().catch((err) => console.log(err));
```

Comment:
I fail to see where this would error, but I'd just bubble it, not silently shallowing the error.